augly/audio/functional.py

```diff
@@ -1269,3 +1269,44 @@ def to_mono(
     )
 
     return audutils.ret_and_save_audio(aug_audio, output_path, sample_rate)
+
+
+def loop_audio(
+    audio: Union[str, np.ndarray],
+    sample_rate: int = DEFAULT_SAMPLE_RATE,
+    n: int = 1,
+    output_path: Optional[str] = None,
+    metadata: Optional[List[Dict[str, Any]]] = None,
+) -> Tuple[np.ndarray, int]:
+    """
+    Takes in an audio file/numpy array and returns it looped a given n number of times
+    @param audio: the path to the audio or a variable of type np.ndarray that
+        will be augmented
+    @param n: the number of times the video will be looped
+    @param output_path: the path in which the resulting audio will be stored. If None,
+        the resulting np.ndarray will still be returned
+    @param metadata: if set to be a list, metadata about the function execution
+        including its name, the source & dest duration, sample rates, etc. will be
+        appended to the inputted list. If set to None, no metadata will be appended
+    @returns: the augmented audio array
```

Comment:
```suggestion
    Loops the audio 'n' times

    @param audio: the path to the audio or a variable of type np.ndarray that
        will be augmented
    
    @param sample_rate: the audio sample rate of the inputted audio

    @param n: the number of times the video will be looped

    @param output_path: the path in which the resulting audio will be stored. If None,
        the resulting np.ndarray will still be returned

    @param metadata: if set to be a list, metadata about the function execution
        including its name, the source & dest duration, sample rates, etc. will be
        appended to the inputted list. If set to None, no metadata will be appended

    @returns: the augmented audio array and sample rate
```
making the docstring more consistent with others in the library (formatting & language-wise) and adding in missing params